### DISCUSSION TITLE
DeltaError: Invalid table version error

### DISCUSSION THREAD

**Discussion by ashishkaul01** [delta-io/delta-rs#3056] [Q&A] [answered]
I have an error occurring when i call the `load_cdf` function with the columns parameter only
`delta_table.load_cdf(columns=['_commit_version'])`
Prior to calling load_cdf i am able to load the delta_table properly. I have verified that.

The error is:
> `File \"/usr/local/lib/python3.11/site-packages/deltalake/table.py\", line 693, in load_cdf\n    return self._table.load_cdf(\n_internal.DeltaError: Invalid table version: 0\n", "level": "ERROR", "level_value": 40,`

Would someone have an idea as to why that might be happening?

**Comment by rtyler** [comment]
@ashishkaul01 Table version zero is typically when the table has just been initialized and the CDF loading code might not handle that properly. Does the table have data in it?

**Comment by ion-elgreco** [comment, marked as answer]
This is resolved in the latest release 0.25+.
